src/main/app/initialize.ts

```diff
@@ -402,9 +402,15 @@ function initializeAfterAppReady() {
         }
 
         const callsWidgetWindow = WindowManager.callsWidgetWindow;
-        if (callsWidgetWindow && webContents.id === callsWidgetWindow.win.webContents.id) {
-            callback(true);
-            return;
+        if (callsWidgetWindow) {
+            if (webContents.id === callsWidgetWindow.win.webContents.id) {
+                callback(true);
+                return;
+            }
+            if (callsWidgetWindow.popOut && webContents.id === callsWidgetWindow.win.webContents.id) {
```

Comment:
Tested it, it looks like it needs to be:
```suggestion
            if (callsWidgetWindow.popOut && webContents.id === callsWidgetWindow.popOut.webContents.id) {
```